src/testConfigs/.bluebutton-config.json

```diff
@@ -1,7 +1,15 @@
 {
-  "clientId": "foo",
-  "clientSecret": "bar",
-  "callbackUrl": "https://www.fake.com/",
-  "version": "1",
-  "environment": "PRODUCTION"
-}
+    "clientId": "foo",
+    "clientSecret": "bar",
+    "callbackUrl": "https://www.fake.com/",
+    "version": "1",
+    "environment": "PRODUCTION",
+    "pkce": true,
+    "retryConfig": {
```

Comment:
I don't think we should be adding this to the config. I think it should just be doing the retry quietly under the covers. I don't think we need to make it completely configurable by the 3rd party app.

Reply:
if that is preferred, this can be removed, I am not sure the default set of retry parameters would fit all situations though.